src/patternfly/components/DataList/examples/DataList.md

```diff
@@ -658,6 +658,124 @@ When a list item includes more than one block of content, it can be difficult fo
+          http://thisisaverylongurlthatneedstobreakusethebreakwordmodifier.org
+        {{/data-list-cell}}
+        {{#> data-list-cell}}
+          http://thisisaverylongurlthatneedstobreakusethebreakwordmodifier.org
+        {{/data-list-cell}}
+        {{#> data-list-cell}}
+          http://thisisaverylongurlthatneedstobreakusethebreakwordmodifier.org
+        {{/data-list-cell}}
+      {{/data-list-item-content}}
+    {{/data-list-item-row}}
+    {{#> data-list-item-row data-list-item-row--modifier="pf-m-nowrap"}}
+      {{#> data-list-item-content}}
+        {{#> data-list-cell}}
+          <span id="{{data-list--id}}-item3">
+           This text will not break or wrap.
+          </span>
+        {{/data-list-cell}}
+        {{#> data-list-cell}}
+          This text will not break or wrap.
+        {{/data-list-cell}}
+        {{#> data-list-cell}}
+          This text will not break or wrap.
+        {{/data-list-cell}}
+        {{#> data-list-cell}}
+          This text will not break or wrap.
+        {{/data-list-cell}}
+      {{/data-list-item-content}}
+    {{/data-list-item-row}}
+  {{/data-list-item}}
+{{/data-list}}
+```
+
+```hbs title=Text-modifiers-data-list-text
+{{#> data-list data-list--id="data-list-with-text-modifiers-and-text" data-list--attribute='aria-label="Data list with modifiers and text"'}}
+  {{#> data-list-item data-list-item--attribute=(concat 'aria-labelledby="' data-list--id '-item1"')}}
+    {{#> data-list-item-row}}
+      {{#> data-list-item-content}}
+        {{#> data-list-cell}}
+          <span id="{{data-list--id}}-item1">
+            This text will wrap to the next line because it has the default behavior of the data list cell.
+          </span>
+          {{#> data-list-text data-list-text--modifier="pf-m-truncate"}}
+            This is data list text, you can apply `pf-m-truncate` directly to the text.
+          {{/data-list-text}}
+        {{/data-list-cell}}
+        {{#> data-list-cell}}
+          This text will wrap to the next line because it has the default behavior of the data list cell.
+          {{#> data-list-text data-list-text--modifier="pf-m-break-word"}}
+            http://thisisaverylongdatalisttextthatneedstobreakusethebreakwordmodifier.org
+          {{/data-list-text}}
+        {{/data-list-cell}}
+        {{#> data-list-cell}}
+          This text will wrap to the next line because it has the default behavior of the data list cell.
+          {{#> data-list-text data-list-text--modifier="pf-m-nowrap"}}
+            This is data list text, you can apply `pf-m-nowrap` directly to the text.
+          {{/data-list-text}}
+        {{/data-list-cell}}
+      {{/data-list-item-content}}
+    {{/data-list-item-row}}
+  {{/data-list-item}}
+{{/data-list}}
+```
+
+### Usage
+| Class | Applied to | Outcome |
+| -- | -- | -- |
+| `.pf-c-data-list__text` | `<span>` | Inserts the data list text element. Use this class to modify specific text directly. |
```

Comment:
We should document that this applies to `*`, which will also let react know it's a flexible element that can be modified with a prop (`component`, I think is what's typically used) to use a different element. It's documented as `*` in a table, too.